projects/demo-integrations/cypress/tests/kit/number/number-decimal-zero-padding.cy.ts

```diff
@@ -1,230 +1,264 @@
+                        .type('{moveToEnd}')
+                        .type('{leftArrow}'.repeat(3))
+                        .type('{backspace}')
+                        .should('have.value', '0,0000')
+                        .should('have.prop', 'selectionStart', '0,'.length)
+                        .should('have.prop', 'selectionEnd', '0,'.length);
+                });
             });
 
-            it('0,000|0 => Delete => 0,0000|', () => {
-                cy.get('@input')
-                    .type('{moveToStart}')
-                    .type('{rightArrow}'.repeat('0,000'.length))
-                    .type('{del}')
-                    .should('have.value', '0,0000')
-                    .should('have.prop', 'selectionStart', '0,0000'.length)
-                    .should('have.prop', 'selectionEnd', '0,0000'.length);
+            describe('Via `Delete` button', () => {
+                it('0,|0000 => Delete => 0,0|000', () => {
+                    cy.get('@input')
+                        .type('{moveToStart}')
+                        .type('{rightArrow}'.repeat('0,'.length))
+                        .type('{del}')
+                        .should('have.value', '0,0000')
+                        .should('have.prop', 'selectionStart', '0,0'.length)
+                        .should('have.prop', 'selectionEnd', '0,0'.length);
+                });
+
+                it('0,0|000 => Delete => 0,00|00', () => {
+                    cy.get('@input')
+                        .type('{moveToStart}')
+                        .type('{rightArrow}'.repeat('0,0'.length))
+                        .type('{del}')
+                        .should('have.value', '0,0000')
+                        .should('have.prop', 'selectionStart', '0,00'.length)
+                        .should('have.prop', 'selectionEnd', '0,00'.length);
+                });
+
+                it('0,00|00 => Delete => 0,000|0', () => {
+                    cy.get('@input')
+                        .type('{moveToStart}')
+                        .type('{rightArrow}'.repeat('0,00'.length))
+                        .type('{del}')
+                        .should('have.value', '0,0000')
+                        .should('have.prop', 'selectionStart', '0,000'.length)
+                        .should('have.prop', 'selectionEnd', '0,000'.length);
+                });
+
+                it('0,000|0 => Delete => 0,0000|', () => {
+                    cy.get('@input')
+                        .type('{moveToStart}')
+                        .type('{rightArrow}'.repeat('0,000'.length))
+                        .type('{del}')
+                        .should('have.value', '0,0000')
+                        .should('have.prop', 'selectionStart', '0,0000'.length)
+                        .should('have.prop', 'selectionEnd', '0,0000'.length);
+                });
             });
         });
     });
+
+    describe('[decimalZeroPadding] is compatible with [postfix]', () => {
```

Comment:
All tests above already existed before (just nested `describe(...) was added`).

All new tests are inside this `describe`.